quickjs.c

```diff
@@ -42189,14 +42189,29 @@ static JSValue js_math_random(JSContext *ctx, JSValueConst this_val,
     return __JS_NewFloat64(ctx, u.d - 1.0);
 }
 
+static double qjs_ceil(double x)
+{
+	return ceil(x);
```

Comment:
Indentation should be 4 spaces for consistency (here and below).

Reply:
done